src/Fetch.js

```diff
@@ -5,7 +5,7 @@
  */
 
 var Fetch = {
-  xhrs: [],
+  xhrs: {},
```

Comment:
Maybe add a comment here?  Something like "Map of fetch ID (Interger) to XHR request object"